frontend/wallet/README.md

```diff
@@ -14,26 +14,30 @@ The Coda Wallet is written in Reason (https://reasonml.github.io/), and built wi
 
 ### Setup
 
-First build the app:
+First set up dependencies
+
+1. Install watchman globally: `brew install watchman`
+2. Install git lfs: `brew install git-lfs`
+3. Run `git lfs install` to update hooks
+
+Download and build the app:
 
 1. Clone the repo via SSH: `git clone git@github.com:CodaProtocol/coda.git`
 2. Navigate into coda/frontend/wallet
 3. Update submodules: `git submodule update --init`
 4. `yarn install` to install dependencies
```

Comment:
nit: maybe we can add (alternatively `npm install`) here as well

will there be any issue with packages if folks try to use npm vs yarn?

Reply:
currently there are some issues involving npm + graphql-faker